### PR TITLE
Updated issue and PR templates for this repo

### LINKED ISSUE #10: Update issue/PR templates for docs repo
## 🧰 Task

It seems like issue/PR templates can be adjusted for repos in an organisation: https://docs.github.com/en/enterprise-server@3.3/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

Currently for issues we are using the same as napari, which includes 'Bug Report', 'Documentation' etc. Potentially we could just have one template?
Also the PR template could be made better suited for documentation.

cc @jni

### PR DESCRIPTION
**Pull request by melissawm**
# Description

Creates specific issue/PR templates for this repo. I have also included links to the other relevant repos (napari/napari and napari/napari-sphinx-theme)

# References
Closes #10 

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,29 +1,20 @@
 # Description
 <!-- What does this pull request (PR) do? Why is it necessary? -->
-<!-- Tell us about your new feature, improvement, or fix! -->
-<!-- If your change includes user interface changes, please add an image, or an animation "An image is worth a thousand words!" -->
+<!-- Tell us about your new content, improvement, or fix! -->
+<!-- If your can, please add an image, or an animation "An image is worth a thousand words!" -->
```
Comment (brisvag) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
<!-- If you can, please add an image, or an animation "An image is worth a thousand words!" -->
```

**Review by brisvag — APPROVED** [review]
LGTM!

**Review by psobolewskiPhD — APPROVED** [review]

**Comment by Czaki** [comment]
It will be nice to request for URL in bug report telmplate

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,29 +1,16 @@
...
-
-## To Reproduce
-
-Steps to reproduce the behavior:
-
-1.
-2.
-3.
-
+<!-- A clear and concise description of what the issue is. -->
```
Comment (psobolewskiPhD) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]:
```suggestion
<!-- A clear and concise description of what the issue is. -->
<!-- Please provide the URL(s) of the page(s) that are problematic. -->
<!-- If you can, please add a screenshot or screen capture: "An image is worth a thousand words!" -->
```

**Comment by psobolewskiPhD** [comment]
> It will be nice to request for URL in bug report telmplate

Actually, yea I think @Czaki is right. 
@melissawm How about adding:
https://github.com/napari/docs/pull/36/files#r1033570253

**Comment by melissawm** [comment]
Good call, thanks!

**Comment by psobolewskiPhD** [comment]
Going to merge. We can then iterate further if needed.

**Pull request merged**